pkg/morph/client/notary.go

```diff
@@ -176,7 +175,7 @@ func (c *Client) DepositNotary(amount fixedn.Fixed8, delta uint32) (res util.Uin
 		big.NewInt(int64(amount)),
 		[]interface{}{c.acc.PrivateKey().GetScriptHash(), till})
 	if err != nil {
-		if !errors.Is(err, neorpc.ErrAlreadyExists) {
+		if !alreadyExistsError(err) {
```

Comment:
Why didn't it work the old way?

Reply:
that is a server error, i did not find any code that returns _exactly_ that err on the client side, did i miss anything?

Reply:
I think it doesn't matter here https://github.com/nspcc-dev/neo-go/blob/master/pkg/neorpc/errors.go#L131

Reply:
in fact, it seems like an unreachable code, cause all the TXs have their own `nonce`, i checked that

so ok, just dropped that commit